frontend/src/app/components/Leaderboards/LeaderboardsRow.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { withRouter } from 'react-router-dom'
+import {
+  TextSpan,
+  ListItem,
+  StyledLink,
+} from '../../../styles/blocks'
+import { createProfilePath } from '../../const/routes'
+
+const LeaderboardsRowComponent = ({ user, position, points }) => {
+  console.log(user)
```

Comment:
pls don't log things

Reply:
Sorry about that, I overlooked it. Will fix it!

Reply:
done [here](https://github.com/salsita/foosball-rating/pull/92/commits/0ca14d1caaa35f00ce4324e704b6cb7f3cf88718#diff-32d0948dd609721b4bda2c94363a08eb)